VideoLocker/build.gradle

```diff
@@ -265,6 +265,12 @@ android {
         def platformName = config.get('PLATFORM_NAME')
         resValue "string", "platform_name", platformName
 
+        def phoneticPlatformName = config.get('PHONETIC_PLATFORM_NAME')
+        if (phoneticPlatformName == null) {
+            phoneticPlatformName = platformName
+        }
+        resValue "string", "phonetic_platform_name", phoneticPlatformName
```

Comment:
If the phonetic platform name is absent, we default the to the platform name

resValue "string", "phonetic_platform_name", phoneticPlatformName actually injects the string where approriate